course4/homework1/test.cc

```diff
@@ -22,6 +22,9 @@ int main()
 	s << a;
 
 	assert(s.str() == "(5, 6)");
+	
+	double pi = 3.14159265358979323846;
```

Comment:
Compiler complains that M_PI is not defined, both if i include cmath or math.h - also using the obscure #define _USE_MATH_DEFINES before. This is with gcc 5.4.0